src/extension/intellisenseHelper.ts

```diff
@@ -182,7 +182,7 @@ export class IntellisenseHelper {
             let setEnvironmentVariableCommand: string = "";
             if (os.type() === "Darwin") {
                 setEnvironmentVariableCommand = "launchctl setenv VSCODE_TSJS 1";
-            } else if (os.type() === "Windows") {
+            } else if (os.type() === "Windows_NT") {
```

Comment:
Is there an advantage to using os.type() instead of os.platform() (which returns "win32" for Windows)?